prompt-service/src/unstract/prompt_service/utils/request.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from enum import Enum
+from typing import Any, Optional
+
+import requests as pyrequests
+from requests.exceptions import RequestException
+
+logger = logging.getLogger(__name__)
+
+
+class HTTPMethod(str, Enum):
+    GET = "GET"
+    POST = "POST"
+    DELETE = "DELETE"
+    PUT = "PUT"
+    PATCH = "PATCH"
+
+
+def make_http_request(
```

Comment:
NIT: This is a good candidate which can be placed and used from `unstract-core`. Feel free to add a TODO / create a ticket if you're not attempting it in the same PR

Reply:
@chandrasekharan-zipstack Yes, I have reused the method from utils of unstract. Need to check refactor this. Will add it as Techdebt

Reply:
https://zipstack.atlassian.net/browse/UN-1654